docs/current/sdk/go/snippets/private-repositories/main.go

```diff
@@ -0,0 +1,25 @@
+package main
+
+import (
+	"context"
+	"fmt"
+
+	"dagger.io/dagger"
+)
+
+func main() {
+	ctx := context.Background()
+	client, err := dagger.Connect(ctx)
+	if err != nil {
+		panic(err)
+	}
+	defer client.Close()
+
+	// Private repository with a README.md file at the root.
+	readme, err := client.Git("git@private-repository.git").Branch("main").Tree().File("README.md").Contents(ctx)
```

Comment:
That takes more space but it avoids scrolling horizontally to see the code.

```suggestion
	readme, err := client.
	    Git("git@private-repository.git").
	    Branch("main").
	    Tree().
	    File("README.md").
	    Contents(ctx)
```

<img width="801" alt="Screenshot 2022-11-02 at 20 32 25" src="https://user-images.githubusercontent.com/61683879/199642687-4953cd47-1f72-4472-bc49-4802e84eda73.png">